communityfaq.json

```diff
@@ -75,35 +75,35 @@
             },
             {
                 "Question":"How do I make missions and campaigns?",
-                "Answer":"Missions can be created in Fred, our mission editor, and hopefully in the future with qtFRED.  The mission file is also in text format, so experienced mission creators can make edits to existing missions by hand, when needed."
+                "Answer":"Missions can be created in FRED, our mission editor, and hopefully in the future with qtFRED.  A mission file is a human-readable text file, so experienced mission creators can make edits to existing missions via text editor, when needed."
             },
             {
-                "Question":"What is Fred?",
-                "Answer":"Fred is the FReespace EDitor, used in making missions and campaigns for the Freespace Open Engine.  It is based on the original mission editor designed by Volition."
+                "Question":"What is FRED?",
+                "Answer":"FRED is the FReespace EDitor, used in making missions and campaigns for the FreeSpace Open engine.  The first versions of FRED were developed by Volition."
             },
             {
-                "Question":"What is qtFred?",
-                "Answer":"qtFred is an in-development rewrite of Fred using the cross-platform qt framework.  It is about halfway complete."
+                "Question":"What is qtFRED?",
+                "Answer":"qtFRED is an in-development rewrite of FRED using the cross-platform Qt framework.  It is about halfway complete."
             },
             {
-                "Question":"What are sexps?",
-                "Answer":"Sexps are symbolic expressions, the original mission and campaign action language.  Sexps are lisp-like and very powerful. You can also define custom variables and containers with them."
+                "Question":"What are SEXPs?",
+                "Answer":"SEXPs, or s-exps (symbolic expressions), are the building blocks for the gameplay logic in FreeSpace missions and campaigns.  SEXPs are adapted from the language Lisp and are very powerful.  Numerous SEXP operators have been added to the original basic collection implemented by Volition."
```

Comment:
```suggestion
                "Answer":"SEXPs, or s-exps are symbolic expressions. They are the building blocks for the gameplay logic in FreeSpace missions and campaigns.  SEXPs are adapted from the language Lisp and are very powerful.  Numerous SEXP operators have been added to the original basic collection implemented by Volition."
```

Reply:
This suggestion seems a bit tautological; "s-exp" is literally the abbreviation for "symbolic expression".  IMO it makes sense to put the abbreviation expansion in parentheses and use the "are" linking verb to introduce the full definition.

Reply:
Parentheses are not going to add clarity or ease of understanding to the target audience of an FAQ.